data/clients.json

```diff
@@ -220,6 +220,26 @@
 			"url": "https://hub.docker.com/r/neilpang/letsproxy",
 			"category": "Docker"
 		},
+	        {
+			"name": "docker-openresty",
+			"url": "https://github.com/xiaojun207/docker-openresty⁠",
+			"category": "Docker",
+			"comments": "(这是一个可以自动申请（并自动更新）免费ssl证书的openresty镜像。This is a Openresty image with auto ssl，use acme.sh)",
```

Comment:
We have a translations system for providing documentation pages in other languages. This description should remain in English so the translators can translate it to all of our supported languages.